pages/embed.tsx

```diff
@@ -31,8 +37,11 @@ const EmbedPage = ({ config, data, error }: IEmbedPageProps) => {
 
 export const getServerSideProps: GetServerSideProps<IPageProps> = async ({
   query,
+  req,
   res
 }) => {
+  log.info({ req }, 'Embed Request');
```

Comment:
I am seeing the `cookie`s coming through in this... and it does contain some PRX sessions for other apps.  At least for localhost.  Can you just remove that?

Reply:
So just drop all cookies? Or some cookies? Which cookies?

Reply:
I'd say all - we don't need to keep that info around in our logs.  (Trying to imagine a case where it would be helpful... maybe if we hit one, we revisit).

Reply:
Pushed up a change to redact cookie keys in JSON output.